Visual Studio Project Template C#/PluginInfrastructure/IScintillaGateway.cs

```diff
@@ -34,40 +34,40 @@ public interface IScintillaGateway
 
         /* ++Autogenerated -- start of section automatically generated from Scintilla.iface */
         /// <summary>Add text to the document at current position. (Scintilla feature 2001)</summary>
-        unsafe void AddText(int length, string text);
```

Comment:
'long'(64-bits) type is much flexible than 'int'(32 bits/64 bits) in both x86 and x64. 

Keeps the interface clean without requiring much changes due to change in data type.